livekit-plugins/livekit-plugins-deepgram/livekit/plugins/deepgram/stt.py

```diff
@@ -123,18 +124,45 @@ def __init__(
         endpointing_ms: int = 25,
         # enable filler words by default to improve turn detector accuracy
         filler_words: bool = True,
-        keywords: list[Tuple[str, float]] = [],
+        keywords: list[Tuple[str, float]] | None = None,
+        keyterms: list[str] | None = None,
```

Comment:
There is no boost float for keyterms? otherwise lgtm!

Reply:
no it's just a list of terms